app/brave_main_delegate.cc

```diff
@@ -191,6 +192,9 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       features::kWebUIDarkMode.name,
       blink::features::kPrefetchPrivacyChanges.name,
       features::kReducedReferrerGranularity.name,
+#if defined(OS_WIN)
+      features::kWinrtGeolocationImplementation.name,
```

Comment:
Looks like it is disabled by default in Chromium, is it safe for us to enable it for the whole userbase?

Reply:
I had a same concern after seeing this comment - https://bugs.chromium.org/p/chromium/issues/detail?id=968883#c33
However, I assume that edge already uses it in stable.

Reply:
Great call on testing w/ Edge 😄